src/controller/python/ChipDeviceController-BlePlatformDelegate.h

```diff
@@ -0,0 +1,67 @@
+/*
+ *
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2014-2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#ifndef DEVICEMANAGER_BLEPLATFORMDELEGATE_H_
+#define DEVICEMANAGER_BLEPLATFORMDELEGATE_H_
+
+#include <ble/BleLayer.h>
+#include <ble/BlePlatformDelegate.h>
+
+using namespace chip::Ble;
+
+extern "C" {
+typedef bool (*WriteBleCharacteristicCBFunct)(void * connObj, void * svcId, void * charId, void * buffer, uint16_t length);
+typedef bool (*SubscribeBleCharacteristicCBFunct)(void * connObj, void * svcId, void * charId, bool subscribe);
+typedef bool (*CloseBleCBFunct)(void * connObj);
+}
+
+class DeviceController_BlePlatformDelegate : public chip::Ble::BlePlatformDelegate
+{
+public:
+    BleLayer * Ble;
```

Comment:
do these need to be public? 
There are setters for callbacks, callers should not need to know the underlying BLE layer.

Reply:
just put it in private, thanks!